src/twisted/internet/interfaces.py

```diff
@@ -1340,7 +1347,7 @@ class IReactorFromThreads(Interface):
     @since: 15.4
     """
 
-    def callFromThread(callable, *args, **kw):
+    def callFromThread(callableObj, *args, **kw):
```

Comment:
I think that this should be named just `f` ... but in a separate PR

```
$ grep -r "def callFromThread(" src/
src/twisted/web/test/test_wsgi.py:    def callFromThread(self, f, *a, **kw):
src/twisted/web/test/test_wsgi.py:            def callFromThread(self, f, *a, **kw):
src/twisted/internet/test/test_base.py:    def callFromThread(self, f, *args, **kwargs):
src/twisted/internet/test/test_resolver.py:        def callFromThread(self, f, *a, **k):
src/twisted/internet/interfaces.py:    def callFromThread(callable, *args, **kw):
src/twisted/internet/asyncioreactor.py:    def callFromThread(self, f, *args, **kwargs):
src/twisted/internet/base.py:        def callFromThread(self, f, *args, **kw):
src/twisted/internet/base.py:        def callFromThread(self, f, *args, **kw):
```

Reply:
I've moved this to: https://github.com/twisted/twisted/pull/1325